src/cli/tc.rs

```diff
@@ -60,51 +59,81 @@ pub fn run() -> Result<()> {
 }
 
 pub async fn init(file_path: &str) -> Result<()> {
-  let tailcallrc: resource::Resource<str> = resource_str!("examples/.tailcallrc.graphql");
-
-  let answer = Confirm::new("Do you want to add a file to the project?")
-    .with_default(false)
-    .prompt();
-
-  match answer {
-    Ok(true) => {
-      let file_name = inquire::Text::new("Enter the file name:")
-        .with_default(".graphql")
-        .prompt()
-        .unwrap_or_else(|_| String::from(".graphql"));
-
-      let file_name = format!("{}.graphql", file_name.strip_suffix(".graphql").unwrap_or(&file_name));
-
-      let confirm = Confirm::new(&format!("Do you want to create the file {}?", file_name))
-        .with_default(false)
-        .prompt();
-
-      match confirm {
-        Ok(true) => {
-          fs::write(format!("{}/{}", file_path, &file_name), "")?;
-
-          let graphqlrc = format!(
-            r#"|schema:
-               |- './{}'
-               |- './.tailcallrc.graphql'
-          "#,
-            &file_name
-          )
-          .strip_margin();
-          fs::write(format!("{}/.graphqlrc.yml", file_path), graphqlrc)?;
-        }
-        Ok(false) => (),
-        Err(e) => return Err(e.into()),
+  let folder_exists = fs::metadata(file_path).is_ok();
+
+  if !folder_exists {
+    let confirm = Confirm::new(&format!("Do you want to create the folder {}?", file_path))
+      .with_default(false)
+      .prompt();
```

Comment:
```suggestion
      .prompt()?;
```

and simplify conditions below